tests/js/spec/components/stackedBarChart.spec.jsx

```diff
@@ -61,22 +61,4 @@ describe('StackedBarChart', function() {
       expect(columns.at(2).text()).toEqual('last seen');
     });
   });
-  describe('functions', function() {
```

Comment:
I removed these tests because they seemed like they were duplicates of the tests I added to dates.spec.jsx. I can see the value of testing at multiple levels, though.

Reply:
IMO there's a small difference. These tests check the template of the string, while the tests in `dates.spec.jsx` check that the getter method is reading from the `ConfigStore`.

You could mock `use24hours` here if you want to be strict on unit-testing.

Reply:
Is there an easy way to mock it out now that it's a util function instead of a method on the wrapper?

Reply:
Yup, Jest has a way to mock imported default/named methods.

https://jestjs.io/docs/en/mock-functions.html#mocking-modules

Reply:
Let me know if you get stuck. I can jump on a call to help.

Reply:
Seems like this would require a lot of workarounds (if it's even possible?) to mock a single function rather than an entire module: https://github.com/facebook/jest/issues/936

Reply:
Ok, I'm going to give up on mocking for now. I'm going to add a getTimeFormat util function, and test that that function returns the correct format string instead of testing all components using that string. Let me know what you think.

Reply:
That's fine. This is not a big deal.